app/styles/_settings.scss

```diff
@@ -17,6 +17,7 @@ $wrap-desktop: 115px;
 // Font sizes
 $font-size-tiny: 10px;
 $font-size-x-small: 12px;
+$font-size-xs-small: 13px;
```

Comment:
honest question: can't we just agree on a standard set of sizes? otherwise this loses total sense